src/main/java/com/adamk33n3r/runelite/watchdog/ui/notifications/panels/NotificationPanel.java

```diff
@@ -50,6 +49,8 @@ public abstract class NotificationPanel extends JPanel {
     static {
         final BufferedImage foregroundImg = ImageUtil.loadImageResource(NotificationPanel.class, "foreground_icon.png");
         final BufferedImage backgroundImg = ImageUtil.loadImageResource(NotificationPanel.class, "background_icon.png");
+        final BufferedImage afkIcon = ImageUtil.loadImageResource(NotificationPanel.class, "afk_icon.png");
+        final BufferedImage nonAFKIcon = ImageUtil.loadImageResource(NotificationPanel.class, "non_afk_icon.png");
```

Comment:
Do you have any thoughts on what the icons should look like? Idk what "afk" looks like haha.

Reply:
Yeah I couldn't come up with an obvious icon. What do you think about an eyeball? Like eye open / eye closed?

Reply:
That's an idea. I had asked in #development on discord and zzz was suggested. I put some screenshots in there. I'm using https://pictogrammers.com/library/mdi/ for icons so if you see something there that would work you can let me know.

Reply:
Ah I didn't realize you were using icon packs. I made these if you like them:

![afk_icon](https://github.com/adamk33n3r/runelite-watchdog/assets/12002072/3fc73494-7451-4826-9557-f868ede8f12a)
![non_afk_icon](https://github.com/adamk33n3r/runelite-watchdog/assets/12002072/94a46872-edb1-44e8-8de7-d281ef19f787)

If not, we could use `eye-closed` and `eye-outline` from the link you posted and just invert the colors.

Reply:
Wow those look so good on this white screen! hahaha, I can't even see them. Finally found them with my mouse to open up :P they look nice, and almost just like the mdi ones! Probably a bit too harsh, like my old icons.

Reply:
Ah I found the non-dark-mode user! Okay I'll take the ones from mdi and resize and color swap them.

Reply:
Not sure what you meant by resize and color swap, but on that page if you go to advanced png download, you can choose the size (16px) and color (white).

Reply:
Ah I didn't see the advanced options. Anyway I already edited it myself and it's up in the latest revision. Shouldn't be distinguishable. 😃